src/qibolab/version.py

```diff
@@ -0,0 +1,3 @@
+import importlib.metadata as im
+
+__version__ = im.version(__package__)
```

Comment:
This we could even leave in `__init__.py` given the simplicity and the fact that it is repeated in other repos, but I don't have a strong opinion.

Reply:
> This we could even leave in `__init__.py` given the simplicity and the fact that it is repeated in other repos, but I don't have a strong opinion.

I would have left there, but we can not, because of circular dependencies.